tpot/base.py

```diff
@@ -632,7 +633,9 @@ def _update_top_pipeline(self, features, target):
                     # Add an extra line of spacing if the progress bar was used
                     if self.verbosity >= 2:
                         print('')
-                    print('Best pipeline: {}'.format(self._optimized_pipeline))
+                    
+                    optimized_pipeline_str = self.clean_pipeline_string(self._optimized_pipeline)
+                    print('Best pipeline:', optimized_pipeline_str)
```

Comment:
Minor issue: Will this still have a space between `Best pipeline:` and the pipeline string?